conanfile.py

```diff
@@ -80,7 +78,24 @@ def imports(self):
         self.copy("license*", src=".", dst="./licenses",
                   folder=True, ignore_case=True)
 
+    def requirements(self):
+        # Direct dependencies:
+        self.requires("fmt/9.1.0")
+        self.requires("nlohmann_json/3.11.2")
+        self.requires("grpc/1.48.0")
+        self.requires("protobuf/3.21.9")
+        self.requires("paho-mqtt-cpp/1.2.0")
+        self.requires("cpr/1.9.3")
+
+        # Transient dependencies:
+        self.requires("openssl/1.1.1t")
+        # Workaround: Pin recipe revision for transient dependency googleapis for enabling the container build
+        self.requires("googleapis/cci.20221108@#e4bebdfa02f3b6f93bae1d5001b8d439")
+        # Workaround: 
```

Comment:
Please add more details in the description comment.
Please provide a ticket when the proper solution is going to be provided.

Reply:
I would like to add a ticket, but the problem is that we currently support bug issues and security issues in GitHub and I don't like to put links to our internal tooling here.